rules/windows/command_and_control_rare_microsoft_office_child_processes.toml

```diff
@@ -0,0 +1,38 @@
+[metadata]
+creation_date = "2023/02/27"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "ES|QL Rule"
+min_stack_version = "8.11.0"
+updated_date = "2023/06/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects rare child processes of Microsoft Office applications, which may indicate an attempt to execute malicious payloads or scripts. This detection focuses on instances where Microsoft Office processes spawn child processes that occur infrequently, potentially as part of a post-exploitation stage of an attack.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
```

Comment:
@Mikaayenson - ES|QL rule types do not allow `index.`

Reply:
I think the real issue is that im inheriting QueryRuleData when it should be just BaseRuleData.

Reply:
Refactored to use the `BaseRuleData` based on the [upstream logic](https://github.com/elastic/kibana/blob/3a18916b31d27d2ffeb7df682e69224a273b39b4/x-pack/plugins/security_solution/server/lib/detection_engine/rule_schema/model/rule_schemas.ts#L167).